src/app/containers/CpsRelatedContent/index.test.jsx

```diff
@@ -51,6 +51,24 @@ describe('CpsRelatedContent', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 
+  it('should render Story Promo components without <ul> when given single item in collection', () => {
+    const topRelatedContentsOneItem = [promos[0]];
+
+    expect(topRelatedContentsOneItem.filter(Boolean).length).toBe(1);
```

Comment:
If this is just null checking `promos[0]`,  it's probably clearer to do it directly
```suggestion
    expect(promos[0]).toBeTruthy();
```